Adding additional links to Serialization (#3915)
JetBrains/kotlin-web-site: docs/topics/serialization.md

As a result, you get a string containing the state of this object in the JSON format: `{"a": 42, "b": "str"}`

> > You can also serialize object collections, such as lists, in a single call.

Reviewer: ```suggestion
> You can also serialize object collections, such as lists, in a single call:
```
Please always add colons after a sentence that you're about to demonstrate with a code sample 😄

> 
> ```kotlin
> val dataList = listOf(Data(42, "str"), Data(12, "test"))
> val jsonList = Json.encodeToString(dataList)

> > ```

Reviewer: ```suggestion
> ```
>
```
As per our style guide, there always needs to be a > on a newline before the curly brackets.

}
```

## What's next

For more information about serialization in Kotlin, see the [Kotlin Serialization Guide](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serialization-guide.md).

> You can further explore various aspects of Kotlin serialization in the following resources:

Reviewer: ```suggestion
You can explore different aspects of Kotlin serialization in the following resources:
```
Shortening for conciseness.

> * [Learn more about Kotlin serialization and its core concepts.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/basic-serialization.md)

Reviewer: Can you remove the full stops at the end of these sentences since they are all links? This is something we need to add to our style guidelines.

* [Explore the built-in serializable classes of Kotlin.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/builtin-classes.md)

> * [Look at serializers in more detail and see how to create custom serializers.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serializers.md)

Reviewer: ```suggestion
* [Look at serializers in more detail and learn how to create custom serializers.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serializers.md)
```

* [Discover how polymorphic serialization is handled in Kotlin.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/polymorphism.md#open-polymorphism)
* [Look into the various JSON features handling Kotlin serialization.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/json.md#json-elements)

> * [Learn more about the various experimental serialization formats supported by Kotlin.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/formats.md)

Reviewer: ```suggestion
* [Learn more about the experimental serialization formats supported by Kotlin.](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/formats.md)
```
